package.json

```diff
@@ -247,6 +247,7 @@
     "eslint-eslines": "0.0.3",
     "eslint-plugin-react": "6.4.1",
     "eslint-plugin-wpcalypso": "3.3.0",
+    "find-root": "1.1.0",
```

Comment:
Don't forget to update the shrinkwrap. This will error out in docker atm with `Cannot find module 'find-root'`